massa-bootstrap/Cargo.toml

```diff
@@ -61,4 +61,8 @@ testing = [
     "massa_ledger_worker/testing",
     "massa_async_pool/testing",
 ]
-sandbox = ["massa_async_pool/sandbox", "massa_final_state/sandbox"]
+sandbox = [
+    "massa_async_pool/sandbox",
+    "massa_final_state/sandbox",
+    "massa_proof_of_stake_exports/sandbox",
```

Comment:
Need to add all dependency that use sandbox such as massa_models

Reply:
done